tabrepo/repository/evaluation_repository.py

```diff
@@ -315,6 +316,7 @@ def evaluate_ensemble(
         datasets: List[str],
         configs: List[str] = None,
         *,
+        ensemble_method_name: str = "greedy ensemble selection",
```

Comment:
Does it seem like a good idea to have strings defined like this ?
do not use spaces